styles.css

```diff
@@ -1,3 +1,20 @@
 body {
     background: #808080;
+}
+.search>input{
+    width: 400px;
```

Comment:
add a max-width: 100% or something so that it works on mobiles with width < 400px